### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/fastertransformer_backend/commit/5318735045741729149506b179717d29417687f0#commitcomment-80554068

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/fastertransformer_backend/commit/46e8f37453dc4a8450cc005f224b0ec223effaf4

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: all_models/gpt/postprocessing/1/utils/gpt_token_encoder.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def __init__(self, encoder, bpe_merges, errors='replace'):
         self.bpe_ranks = dict(zip(bpe_merges, range(len(bpe_merges))))
         self.cache = {}
 
-        # Should haved added re.IGNORECASE so BPE merges can happen for capitalized versions of contractions
+        # Should have added re.IGNORECASE so BPE merges can happen for capitalized versions of contractions
```
Comment (jsoref) [inline all_models/gpt/postprocessing/1/utils/gpt_token_encoder.py:90]: This was manual...

File: docs/gpt_guide.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ python ft_workspace/fastertransformer_backend/tools/end_to_end_test.py
 
 after launching the triton server.
 
-Regarding `bad_words_dict` and `stop_words_dict`, they should provide a single CSV-formatted string per item. The string then reprensent a list of words or expressions and each element is tokenized for further use by the model. Beware of tokenizer subtleties, for example, "word" and " word" are two distinct tokens. You can use the script in `all_models/gpt/preprocessing/1/word_list.py` to help you understand the tokenization.
+Regarding `bad_words_dict` and `stop_words_dict`, they should provide a single CSV-formatted string per item. The string then represent a list of words or expressions and each element is tokenized for further use by the model. Beware of tokenizer subtleties, for example, "word" and " word" are two distinct tokens. You can use the script in `all_models/gpt/preprocessing/1/word_list.py` to help you understand the tokenization.
```
Comment (jsoref) [inline docs/gpt_guide.md:171]: This should probably be `represents`

**Review by jsoref — COMMENTED** [review]
Most corrections were automatically suggested by Google Sheets. All fault mine.

**Comment by byshiue** [comment]
Thank you for the modification.

**Pull request merged**